src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -2132,4 +2132,19 @@ Widening and user defined conversions are not supported with generic types.</val
   <data name="DoNotPassNullableStructToArgumentNullExceptionThrowIfNullCodeFixTitle" xml:space="preserve">
     <value>Replace the 'ArgumentNullException.ThrowIfNull' call with a conditional</value>
   </data>
+  <data name="DoNotCompareSpanToNullDescription" xml:space="preserve">
+    <value>Because the 'null' literal will be implicitly converted to a span, comparing a span to 'null' might not do what you intended to check. Make it more explicit by comparing to 'default' or by using 'IsEmpty'.</value>
```

Comment:
```suggestion
    <value>Comparing a span to 'null' might not do what you intended because the 'null' literal will be implicitly converted to a span. Make the code more explicit by comparing to 'default' or using 'IsEmpty'.</value>
```

Reply:
Again, removing the `default` part from @gewarren's suggestion
```suggestion
    <value>Comparing a span to 'null' might not do what you intended because the 'null' literal will be implicitly converted to a span. Remove the comparison as struct will never be 'null' or explicitly use 'IsEmpty' if that was the intention.</value>
```